.github/pull_request_template.md

```diff
@@ -9,8 +9,9 @@ refactor/make-loop-more-better
 “fix: Do the thing”
 
```

Comment:
Going forward we should probably remove the branch and commit message template from the pull request.